CONTRIBUTING.md

```diff
@@ -1,44 +1,63 @@
 # Contributing Guidelines
 
-Hi! Nice to have you with us. This file will walk you through the steps you need to take if you want to contribute a word or some code to the project. Let’s go.
+Hi! Nice to have you with us. This guide will walk you through the steps you have to take if you want to contribute a word or some code to the project. Let’s go.
 
-Actually – before we start – a quick word of caution: unfortunately, the contribution process involves some technical knowledge. We are aware that this poses a problem for most people and are discussing how a more inclusive contribution process could look like.
+## Before You Start
+
+The contribution process currently relies on some knowledge of working with Github and writing in Markdown. We understand that this creates a barrier to entry for folks who haven't worked with either of those before. Long term, we hope to provide a solution that can minimise these sorts of barriers.
+
+We also want to encourage you to dive into Github and Markdown even if you haven't before. We've collected a list of [useful links](#useful-links) that may help you understand both Github and Markdown:
+
+- [Chaser Pettit's introduction to the GitHub workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
+- [Basic Markdown Syntax](https://www.markdownguide.org/basic-syntax/)
+- [Good First Issues](https://github.com/tatianamac/selfdefined/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22good+first+issue%22+)
+
+Please remember that the contributing team is always here to help. No question is "too simple" or "too easy". We'd rather you ask, as someone else might want to know, too!
```

Comment:
I commented this on the previous commit (sometimes I get lost in multiple comments 😬), but we can add maybe a "Where to Get Help" section broken out? If you think that's overkill for right now, that's fine. I wrote a bit in case helpful:

> Where to Get Help
> If you've already tried reading through our [documentation](https://www.selfdefined.app/documentation/) and are stuck, we're here to help.
> 
> You can contact us/ask questions by reaching out to us on Twitter at [@SelfDefinedApp](https://www.twitter.com/selfdefinedapp).
> You can file an issue if you think some information is missing and/or if it would be helpful.

Reply:
I'll add it, but I also think it is a good practice to put "get help" info into README as well.